### ISSUE TITLE
Standard format for examples

### ISSUE THREAD

**Issue by LandonSchropp** [docopt/docopt#449]
Is there a standard format for adding examples to the documentation. Some of our commands can get a little complex, and it's nice to print examples that can be copied pasted.

```
Naval Fate.

Usage:
  naval_fate ship new <name>...
  naval_fate ship <name> move <x> <y> [--speed=<kn>]
  naval_fate ship shoot <x> <y>
  naval_fate mine (set|remove) <x> <y> [--moored|--drifting]
  naval_fate -h | --help
  naval_fate --version

Options:
  -h --help     Show this screen.
  --version     Show version.
  --speed=<kn>  Speed in knots [default: 10].
  --moored      Moored (anchored) mine.
  --drifting    Drifting mine.

Examples:
  Here's how you'd move the ship `Disney Cruise` to the coordinates `<2, 3>`:

  naval_fate ship "Disney Cruise" move 2 3 --speed=100

  Here's how you'd shoot at the coordinates `<4, 6>`:

  naval_fate ship shoot 4 6
```

**Comment by Nicholas-Mitchell** [comment]
Adding the examples as you did will show up just as you typed them if someone executes:

```bash
naval_fate --help
```
does that break something for you? It works for me just fine. The format itself is irrelevant, I believe.

You could use the tool [**`tldr`**](https://github.com/tldr-pages/tldr-python-client), where you specify a set of examples and people can just call:

```bash
tldr your_program
```
and the get a nice output. For example, doing so with `tar` in a Linux terminal:

![image](https://user-images.githubusercontent.com/12820901/59964138-e66bba80-94fc-11e9-95d8-56fa30c44de4.png)
